pkg/webui/components/event/types/default/index.js

```diff
@@ -40,7 +39,6 @@ class DefaultEvent extends React.PureComponent {
     const { className, event, widget, overviewClassName, expandedClassName } = this.props
 
     const entityId = getEntityId(event.identifiers[0])
-    const data = formatMessageData(event.data)
```

Comment:
Are you sure there can be no relevant data fetched here? What about `ApplicationUp`?
Just want to be sure.

Reply:
I think this is part of the `Event.Message` component. Since this case was handled before I added the check in `Event.Default`